esmvalcore/_main.py

```diff
@@ -94,14 +94,16 @@ def process_recipe(recipe_file, config_user):
             "NetCDF compression.")
 
     # copy recipe to run_dir for future reference
-    shutil.copy2(recipe_file, config_user['run_dir'])
+    shutil.copy2(recipe_file, os.path.join(config_user['run_dir'],
```

Comment:
Not sure if renaming the recipe will set people off. I think you did this for making it available in the generated html, right? Could we instead glob for something like `recipe_*.yml`? It would also be nice if the user saw the name of the original recipe so they can easily find it if they wanna reproduce.

Reply:
Yeah, that is right. I think it fits with giving everything a standard name inside the output directory, this makes it easy to find. I had to do something here so that running a recipe from the API and the CLI has the recipe in the same place. This is by far the easiest solution, because otherwise I would have to generate the name of the recipe in the session object. I can have another go at that. Globbing will also work, but it seems a bit fragile to me.

The name of the recipe in the output directory is not covered by the tests, which is why I was confident to change it. The name of the original recipe is in the output directory.

Reply:
Thanks for the suggestion, this is fixed in the latest commit 👍